storage/storage.c

```diff
@@ -74,6 +80,17 @@
 // NOTE: This should always equal the value in VERSION_KEY.
 #define UNAUTH_VERSION_KEY ((APP_STORAGE << 8) | 0x08)
 
+#if USE_TROPIC
+// Key that is used to reset the M&D slots in Tropic after successfull unlock.
+#define TROPIC_HMAC_RESET_KEY ((APP_STORAGE << 8) | 0x09)
```

Comment:
```suggestion
#define MAC_AND_DESTROY_RESET_KEY ((APP_STORAGE << 8) | 0x09)
```

Reply:
Renamed to `TROPIC_MAC_AND_DESTROY_RESET_KEY` in https://github.com/trezor/trezor-firmware/pull/5845/commits/867deb6169ef41ef8cae824c1f7260c61ae1864f.